sql_generators/glean_usage/templates/usage_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,108 @@
+{{ header }}
+
+WITH usage_reporting_base AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
+    metrics.uuid.usage_profile_id,
+    normalized_channel,
+    -- client_info.app_display_version,
+    -- client_info.app_build,
+    normalized_os,
+    normalized_os_version,
+    -- client_info.locale,
```

Comment:
@irrationalagent all of the `client_info.*` fields are missing in the new ping. Are we planning on adding those or should I just delete these from the code?

Reply:
Some of these will be added back, not under client_info but likely as "normal" metrics e.g. string metrics. Specifically, we can expect these fields to be added as metrics:

App_version
App_build
first_run_date
reason
OS
Duration
Is default browser
Distribution_id
Channel

I will follow up about where exactly in the ping we will expect to see those.